### ISSUE TITLE
Feature request: Or filter

### ISSUE THREAD

**Issue by ken-mills** [cscan/vue-excel-editor#50]
I realize I can input a custom filter for Or for example `~[dog|cat]`. It would be great if that feature was available in the filter popup by selecting multiple values and then selecting apply. Currently, when I select a single value, the column is only filtered by that single value. Multiple selections is not an option.
Thanks so much.
--Ken

**Comment by cscan** [comment]
Yes, now the dialog just provides single-selection mode. I will think about it.

**Comment by ken-mills** [comment]
I came up with a solution which does not require the user to type a regex expression each time. It appears to work for my use case while still supporting existing filter conditions. The fix opens the dialog with a brief instruction, then requires the user to select a filter condition prior to clicking one or more checkbox options, and then select apply. Let me know if you would like to review a pull request and update the instructions. I called the option:

```
thisOrThat: '| This Or That',
```

![panelFilter dialog 2020-12-11 083512](https://user-images.githubusercontent.com/17298135/101909726-e695c500-3b8b-11eb-8eee-1f2bb668ed70.png)

![panelFilter dialog 2020-12-11 083755](https://user-images.githubusercontent.com/17298135/101909992-599f3b80-3b8c-11eb-8ca5-560b1c0d346a.png)


I added a new "|" filter condition and supporting filter function, case #10 to support uppercase. The checkboxes values are captured using v-model.

...panelFilter.vue

```
v-model="selectedFilters"
.
.
. 
if (this.selectedFilters.length === 1) {
 opt = this.inputFilterCondition + this.selectedFilters[0]
} else {
 if (this.inputFilterCondition === '|') {
   opt = "|" + this.selectedFilters.join("|")
 }
}

```
...VueExcelEditor

```
case this.columnFilter[k].startsWith('|'):
 filter[k] = {type: 10, value: this.columnFilter[k].slice(1).trim().toUpperCase()}
 break
```

**Comment by cscan** [comment]
Let me think about it
